src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -193,10 +199,13 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
-            sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
-                    config.getCucumberOutputDir()
-                    .replace('\\', '/'), fileCounter, formatStr));
-
+            sb.append(String.format("\"%s:%s/%s/%s.%s\"", formatStr,config.getCucumberOutputDir().replace('\\', '/'),FilenameUtils.removeExtension(outputFileName), FilenameUtils.removeExtension(outputFileName), formatStr));
+            if(formatStr.equalsIgnoreCase("html"))
+                htmlFormat =String.format("\"%s:%s/%s/%s.%s\"", formatStr,config.getCucumberOutputDir().replace('\\', '/'),FilenameUtils.removeExtension(outputFileName), FilenameUtils.removeExtension(outputFileName), formatStr);
+            if(formatStr.equalsIgnoreCase("json"))
+                jsonFormat =String.format("\"%s:%s/%s/%s.%s\"", formatStr,config.getCucumberOutputDir().replace('\\', '/'),FilenameUtils.removeExtension(outputFileName), FilenameUtils.removeExtension(outputFileName), formatStr);
+            if(formatStr.equalsIgnoreCase("rerun"))
+                rerunFormat =String.format("\"%s:%s/%s/%s.%s\"", formatStr,config.getCucumberOutputDir().replace('\\', '/'),FilenameUtils.removeExtension(outputFileName), FilenameUtils.removeExtension(outputFileName), formatStr);
```

Comment:
rerun file extension is ".txt", not rerun.

The junit extension is ".xml", not ".junit"

Should line 202 be the "else" case?